packages/postcss-ordered-values/src/lib/isMathFunctionNode.js

```diff
@@ -0,0 +1,13 @@
+import { both, compose, either, includes, prop, toLower } from 'ramda';
+import isFunctionNode from './isFunctionNode';
+import includedIn from './includedIn';
+
+const _isMathFunctionNode = compose(
+  either(includedIn(['min', 'max', 'clamp']), includes('calc')),
```

Comment:
min/max/clamp like a calc function (new spec)

Reply:
So we should handle `-webkit-clamp()` etc?

Reply:
can't find docs about this, but i think will be great always drop prefix

Reply:
OK, will update. 👍